### DISCUSSION TITLE
みんなで開発した時の感想を残しておく記念スレッド

### DISCUSSION THREAD

**Discussion by DaisukeShinoku** [yuki-snow1823/diary#59] [Show and tell]
【名前】

【やったこと】

【やってること】

【感想】

のフォーマットで書いていくやで

**Comment by DaisukeShinoku** [comment]
2023.06.17

セキュリティCREやるホリコシユウキ
【やったこと】フロントエンドをyarnに寄せて、Vercelを剥がし、Netlifyで見れるようにした、レビュー
【やってること】GraphQLをフロント側で使用できるように試行錯誤
【感想】会社だと先人が作ってくれた環境で開発できるので、それを1から作るとなると難しいなと思った…

奈良脩平
【やったこと】
graphqlを使用した記事更新機能をペアプロ
新奥さんとテストの内容について話し合い
【やってること】
特になし！見つけます！
【感想】
今日も楽しかった！綺麗なフォルダ構成や書き方を考えるのとても良きです

シンオク ダイスケ
【やったこと】GraphQLをRSpecする際の最適解を考えていた
【やってること】journalの作成に関するRSpecをサクッと書いてみた、これを土台にみんなとRSpecの正解を模索したい
【感想】RSpecがそもそも知識が乏しいから勉強しないだなあ

ぽこ
【やったこと】
READMEにフロントの構成を追加
ジャーナル投稿機能実装のモブプロみたいなこと
【やってること】
特になし（GraphQLとかReactを学習しよー）
【感想】
普段使わない技術に触れられた
エンジニアと対面でコーディングする機会があまりないのでよい刺激になった
楽しかった（小並感）

かりまた かな
【やったこと】
・環境構築
・README修正
・ログイン機能のview側のソースをfrontのデザインパターンに整える
【やってること】
・ログイン機能のview側のソースをfrontのデザインパターンに整える→引き続き頑張る
【感想】
・React初めてで分からない事多かったけど、楽しかった

イソベコウキ
【やったこと】
journalの更新機能を作成してマージした。
【やってること】
今後の実装のため、フロント側でGraphQLをどう使えるのようにするか一緒に考えている
【感想】
とりあえず機能作成して、マージできたのは嬉しかった。教えてくださった奈良さんとレビューしてくださった堀越さん、ありがとうございました。
フロント側の実装は、正直分からないことが多すぎて現状厳しい。手を動かして理解していくしかないかなと思う。

**Comment by yuki-snow1823** [comment]
@DaisukeShinoku 
ナイス投稿感謝です！！これまたやっていきましょう
